### DISCUSSION TITLE
Alarmauslösung Home App

### DISCUSSION THREAD

**Discussion by Blaulichttourist** [thkl/hap-homematic#344] [Q&A]
Hallo!

Ich bin neu hier und habe direkt eine Frage/ ein Problem, vielleicht könnt ihr mir ja helfen :-) 

Ich habe eine Alarmanlage in HM konfiguriert und diese auch mit HAP-HM versucht in meiner Home App dar zu stellen. Dank der wunderbaren Erklärungen hier, hat es auch wunderbar geklappt, allerdings wird mir der ausgelöste Alarm in meiner Homeapp erst dann angezeigt, wenn ich diese öffne. Wenn sie z.B. im Hintergrund läuft, bekomme ich keine Benachrichtigung, sobald ich die App auf dem Handy öffne, dann sofort auf allen Geräten. 
Ist das normal oder mache ich was falsch?

LG Jan

**Comment by roe1974** [comment]
Die Statusänderungen der Alarm-Sirene (Alarm Abwesenheit, Anwesenheit etc.) bekomsmt Du als Push-Nachricht ?

**Reply by Blaulichttourist** [reply]
Die bekomme ich leider auch erst, wenn ich die Home app geöffnet habt. Aber dann bekommen sie alle Geräte, die in Home eingebunden sind, also auch meine Freundin. 
Trigger habe ich über einen Virtuellen Knopf an der CCU eingebaut und auch im HAP-HM Programm so angegeben.

**Comment by roe1974** [comment]
OK ... ich verwende die "echte" Alarmsirene .. da funktioniert es ohne Probleme... das muss Thomas beantworten
Kann aber wenn nur am Trigger liegen....

**Reply by Blaulichttourist** [reply]
OK, danke schonmal. Was ist denn die "echte"?

**Comment by roe1974** [comment]
In der HM Welt zB. die HM-Sec-Sir-WM
lg Richard

**Reply by Blaulichttourist** [reply]
Ah ok, ja ich benutze auch 2 echte. Die HmIP-ASIR-2 und die HmIP-ASIR-0. Aber die kann man nicht in die Home App einbinden oder?

lg Jan

**Comment by roe1974** [comment]
HmIP-ASIR-0 habe ich auch ... ist aber nur die Aussensirene (macht keinen Sinn die einzubinden)
HmIP-ASIR-2 sollte das "pendant" zur HM-Sec-Sir-WM sein ... die würde ich mal einbinden probieren in HK.
Wenn zweitere nicht geht, dann hier das json file (als txt) der Sirene posten (in HAP über "internes" -> "Hilfe")
lg Richard

**Comment by roe1974** [comment]
OK .. das hatten wir schon:
https://github.com/thkl/hap-homematic/issues/50
Die HmIP-ASIR-2 ist auch "nur" eine Sirene ... also kein "pendant" zur Hm-Sec-Sir-WM wie es aussieht :-(
Bleibt nur der "Nachbau" über die Variablen......

**Comment by Blaulichttourist** [comment]
Ja, dass habe ich auch schon gefunden. Seltsam, seltsam. Bei anderen dingen, wie z.B. die Erkennung von Personen durch den Bewegungsmelder, schickt mit die Home App automatisch Benachrichtigungen. Aber im Alarmfall nicht. Genauso seltsam finde ich, dass wenn ich die Home App dann öffnet, sofort auf allen Geräten eine Push Nachricht kommt, dass ein Alarm ausgelöst wurde...

**Comment by jb-home** [comment]
Wenn das mit Variablen gebaut wurde, dann stimmt wohl etwas an dem Variablen Update Träger nicht.

**Comment by kronika01** [comment]
Ich habe genau das gleiche Problem.

**Comment by Path78** [comment]
Mit Variablen keine Probleme hier
